com.albiononline.AlbionOnline.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Type=Application
+Name=Albion Online
+GenericName=MMORPG
+Comment=Play Albion Online
+Icon=com.albiononline.AlbionOnline
+Exec=/app/bin/albion-online
```

Comment:
No reason. I did not go through a wrapper sscript in the beginning. When I introduced it, I changed an absolute path by another absolute path withoyt thinking. Will fix that once I am home.